src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/announcements/AnnouncementDashboardPage.java

```diff
@@ -21,11 +21,6 @@ public void goTo() {
         driver.get("dashboard");
     }
 
-    public void viewReadAnnouncements() {
-        WebElement viewReadButton = driver.findElement(By.id("view-read-announcements"));
-        viewReadButton.click();
-    }
-
     public List<WebElement> getCurrentUnreadAnnouncements() {
         return driver.findElements(By.cssSelector(".t-announcement-item"));
```

Comment:
Looking through the code, I know this is pretty legacy so I don't want you to change it, but I don't like the "test" to know anything about the page.  I.e. it should not get any WebElements.  You could ask for how many announcements there are or something like that from the test.

Reply:
OK.